src/styles/_summary.scss

```diff
@@ -1,4 +1,6 @@
 .h5p-interactive-book {
+  container-type: inline-size;
+  container-name: summary-page;
```

Comment:
It's good that you gave the container a name! It helps to prevent confusion if containers get nested. But I'll rename it. You're using the container for the summary page, but later on it might be used for something else, too - and then the name would not make sense.